packages/svelte-urql/src/subscriptionStore.ts

```diff
@@ -0,0 +1,59 @@
+import { pipe, subscribe } from 'wonka';
+import type { OperationContext, Client } from '@urql/core';
+import { derived, writable } from 'svelte/store';
+import type { DocumentNode } from 'graphql';
+import type { TypedDocumentNode } from '@graphql-typed-document-node/core';
+import { createRequest } from '@urql/core';
+import type { AnnotatedOperationResult, UrqlStore } from './common';
+import { defaultBaseResult, fetchProcess } from './common';
+
+/**
+ * Create a Svelte store for an [Urql subscription](https://formidable.com/open-source/urql/docs/api/core/#clientexecutemutation) using [Wonka](https://wonka.kitten.sh/)
+ */
+export function subscriptionStore<Data, Variables extends object = {}>(props: {
+  /** an [Urql client](https://formidable.com/open-source/urql/docs/api/core/#client)  */
+  client: Client;
+  /** a graphql tag */
+  subscription: DocumentNode | TypedDocumentNode<Data, Variables> | string;
+  /** the variables to be used in the fetch operation */
+  variables?: Variables;
+  /** Urql fetching options */
+  context?: Partial<OperationContext>;
+  /** the initial result value (prior to fetch) */
+  placeholderData?: Partial<AnnotatedOperationResult<Data, Variables>>;
```

Comment:
same question here, why are we indicating initial data for a subscription?

Reply:
dropped https://github.com/FormidableLabs/urql/pull/2370/commits/df05cd86b6313a41a9eea74df950f3d0b699b8e4